wiki/existing-rules-files.md

```diff
@@ -1,25 +1,23 @@
 # Existing rules
```

Comment:
Mattias commented earlier that titles should be capitilized, does the same apply here?